packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown.tsx

```diff
@@ -93,7 +93,7 @@ export class SelectWrapperDropdown {
       // therefore we do it here via attribute
       ['hidden', 'disabled', 'selected']
     );
-    this.isNativePopoverCase = detectNativePopoverCase(this.host, true);
+    this.isNativePopoverCase = !!detectNativePopoverCase(this.host, true);
```

Comment:
boolean casting shouln't be necessary as `detectNativePopoverCase` returns a boolean

Reply:
`TS2322: Type boolean | void is not assignable to type boolean`
The current implementation of `detectNativePopoverCase` has no default return --> `TS7030: Not all code paths return a value.`